.storybook/webpack.config.js

```diff
@@ -58,6 +59,29 @@ module.exports = ({ config }) => {
 		return name.includes('babel-preset-minify') === false;
 	});
 
+	config.module.rules.push({
+		test: /\.(scss|sass)$/,
+		use: [
+			{
+				loader: require.resolve('style-loader')
+			},
+			{
+				loader: require.resolve('css-loader'),
+				options: {
+					url: false,
+					import: false,
+					modules: true
+				}
+			},
+			{
+				loader: require.resolve('sass-loader'),
+				options: {
+					includePaths: glob.sync('./components/*/bower_components', { absolute: true }),
```

Comment:
nice solution

Reply:
It's a bit of a hack because if two different components depend on different versions of bower things then it could get sticky. but it gets this moving and we can iterate that later.

Reply:
#183 should fix that in the future